### PR TITLE
Remove types

### PR DESCRIPTION
**Pull request by v0dro**
All tests except those that were failing before the change are passing.

### CODE REVIEW

**Comment by agarie** [comment]
Hey Sameer, can you remove the update to History? I think this kind of update should be made on its own. It's easier to browse the commit log that way.

File: History.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+=== 1.5.0
```
Comment (agarie) [inline History.txt]: I'm talking about this change.

**Comment by v0dro** [comment]
ok done. Is everything else alright? Is it ok to start with daru now?

File: lib/statsample/vector.rb [hunk 2]
```diff
@@ -8,9 +8,15 @@ def to_vector(*args)
     Statsample::Vector.new(self,*args)
   end
 
-  # Creates a new Statsample::Vector object of type :scale
+  # Creates a new Statsample::Vector object of type :scale.
+  # Deprecated. Use to_numeric instead.
   def to_scale(*args)
-    Statsample::Vector.new(self, :scale, *args)
+    puts "WARNING: to_scale has been deprecated. Use to_numeric instead."
```
Comment (agarie) [inline lib/statsample/vector.rb]: Use `STDERR.puts` instead please.

**Comment by agarie** [comment]
Almost. Can you provide a test showing that it's possible to create vectors with the old interface? I'll merge this PR with that in place. Also, it's usually a good idea to use `STDERR.puts` instead of plain `puts` when printing warnings. Sometimes it doesn't make a difference, but it can avoid some headaches when printing results in a script for a large-y dataset.

With these comments in place---yes, please, let's get started with Daru! :+1:

**Comment by v0dro** [comment]
ok done.

**Pull request merged**